### ISSUE TITLE
Add is_ascii_letter() to stdlib/ascii.jou

### ISSUE THREAD

**Issue by Akuli** [Akuli/jou#408]
Pretty self-explanatory. Should return True for letters A-Z and a-z.

Needs test in `tests/should_succeed/ascii_test.jou`

While we're at it we might as well add other similar functions, if I forgot any (not sure). We could look at what's in ctype.h for some inspiration.

@littlewhitecloud want to do this? :)

**Comment by littlewhitecloud** [comment]
Just use isalpha :D. I already declared them.
https://github.com/Akuli/jou/blob/ee3bf89f4920f0519c6d48f34c3bb43387068116/stdlib/types.jou
```
# Functions for checking types

# Check if character is alphanumeric
declare isalnum(c: int) -> int
# Check if character is alphabetic
declare isalpha(c: int) -> int
# Check if character is a control character
declare iscntrl(c: int) -> int
# Check if character is decimal digit
declare isdigit(c: int) -> int
# Check if character has graphical representation
declare isgraph(c: int) -> int
# Check if character is lowercase letter
declare islower(c: int) -> int
# Check if character is printable
declare isprint(c: int) -> int
# Check if character is a punctuation character
declare ispunct(c: int) -> int
# Check if character is a white-space
declare isspace(c: int) -> int
# Check if character is uppercase letter
declare isupper(c: int) -> int
# Check if character is hexadecimal digit
declare isxdigit(c: int) -> int
# Convert uppercase letter to lowercase
declare tolower(c: int) -> int
# Convert lowercase letter to uppercase
declare toupper(c: int) -> int
```

**Comment by littlewhitecloud** [comment]
Akuli, maybe we can create a issue and put all things to do and link issue. And then you can put todos and I can choose to do.

**Comment by littlewhitecloud** [comment]
I forget that `<type.h>` already has `int isascii(int c)` function and @Akuli you can use that function instead of write one.

**Comment by littlewhitecloud** [comment]
```python
declare isalpha(_C: int) -> int
declare isupper(_C: int) -> int
declare islower(_C: int) -> int
declare isdigit(_C: int) -> int
declare isxdigit(_C: int) -> int
declare isspace(_C: int) -> int
declare ispunct(_C: int) -> int
declare isalnum(_C: int) -> int
declare isprint(_C: int) -> int
declare isgraph(_C: int) -> int
declare iscntrl(_C: int) -> int
declare toupper(_C: int) -> int
declare tolower(_C: int) -> int
declare __isascii(_C: int) -> int
declare __toascii(_C: int) -> int
```

**Comment by Akuli** [comment]
Fixed in #418
